Explorer/Assets/Scripts/ECS/SceneLifeCycle/SceneFacade/Systems/LoadHibridSceneSystemLogic.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Threading;
+using CommunicationData.URLHelpers;
+using Cysharp.Threading.Tasks;
+using DCL.Diagnostics;
+using DCL.Ipfs;
+using DCL.WebRequests;
+using ECS.Prioritization.Components;
+using ECS.SceneLifeCycle.Components;
+using ECS.SceneLifeCycle.SceneDefinition;
+using SceneRunner;
+using SceneRunner.Scene;
+using UnityEngine;
+using Utility;
+
+namespace ECS.SceneLifeCycle.Systems
+{
+    public class LoadHibridSceneSystemLogic : LoadSceneSystemLogicBase
+    {
+        private readonly string hibridSceneID;
+        private readonly string hibridSceneContentServer;
+
+        public LoadHibridSceneSystemLogic(IWebRequestController webRequestController, URLDomain assetBundleURL, string hibridSceneID, string hibridSceneContentServer) : base(webRequestController, assetBundleURL)
+        {
+            this.hibridSceneID = hibridSceneID;
+            this.hibridSceneContentServer = hibridSceneContentServer;
+        }
+
+        public override async UniTask<ISceneFacade> FlowAsync(ISceneFactory sceneFactory, GetSceneFacadeIntention intention, string reportCategory, IPartitionComponent partition, CancellationToken ct)
+        {
+            var definitionComponent = intention.DefinitionComponent;
+            var ipfsPath = definitionComponent.IpfsPath;
+            var definition = definitionComponent.Definition;
+
+            // Warning! Obscure Logic!
+            // Each scene can override the content base url, so we need to check if the scene definition has a base url
+            // and if it does, we use it, otherwise we use the realm's base url
+            var contentBaseUrl = ipfsPath.BaseUrl.IsEmpty
+                ? intention.IpfsRealm.ContentBaseUrl
+                : ipfsPath.BaseUrl;
+
+            ISceneContent hashedContent = new SceneHashedContent(definition.content, contentBaseUrl);
```

Comment:
The only difference here is how you create `ISceneContent`, everything else is copied, thus I propose to create a single abstract method to provide `ISceneContent` and everything else can be in the base class

Reply:
Done. The methods in `LoadSceneSystemLogic` feel a little bit redudant, but needed to use the abstraction